frontend/Dockerfile

```diff
@@ -2,6 +2,10 @@ FROM node:latest
 
 ENV MAPBOX_TOKEN "$MAPBOX_TOKEN"
 
+# Set up nginx to proxy requests.
+# Should probably switch to express since we are in a node container anyway.
+RUN apt-get update; apt-get -y install nginx
```

Comment:
Could we start Nginx in the docker directly? Do we want to?

Reply:
We don't want it to run on non-remote environments.